core/src/test/java/org/elasticsearch/index/query/AbstractQueryTestCase.java

```diff
@@ -158,11 +157,6 @@
     private static int queryNameId = 0;
     private static SearchModule searchModule;
 
-
-    protected static QueryShardContext queryShardContext() {
-        return queryShardContext;
```

Comment:
I am now wondering if we still need this queryShardContext instance member if we (almost) always create a new one.